src/app/FakeLib/TargetHelper.fs

```diff
@@ -345,13 +382,72 @@ let run targetName =
                     traceEndTarget target.Name                
         with
         | exn -> targetError targetName exn
+
+    let parallelJobs = environVarOrDefault "fake-parallel-jobs" "1" |> Int32.Parse
       
     let watch = new System.Diagnostics.Stopwatch()
     watch.Start()        
     try
         tracefn "Building project with version: %s" buildVersion
-        PrintDependencyGraph false targetName
-        runTarget targetName
+
+        if parallelJobs > 1 then
```

Comment:
The body of this `if` is very long and hard to read. What about extracting a function or two?